tests/cypress/e2e/spec.cy.ts

```diff
@@ -33,4 +34,52 @@ describe('Source configuration', () => {
     deleteCmkHost(hostName);
     activateCmkChanges('cmk');
   });
+
+  it('create a new time-usage panel', { defaultCommandTimeout: 10000 }, () => {
+    createCmkHost(hostName);
+    activateCmkChanges('cmk');
+
+    cy.visit('/');
+    cy.get('input[name="user"]').type(Cypress.env('grafanaUsername'));
+    cy.get('input[name="password"]').type(Cypress.env('grafanaPassword'));
+    cy.get('[aria-label="Login button"]').click();
+
+    cy.visit('/dashboard/new');
+    cy.get('button[aria-label="Add new panel"]').click();
+
+    cy.get('input[id="react-select-7-input"]').type('Host name{enter}'); // Filter -> 'Host name'
+
+    cy.get('input[id="input_Host"]').type('{enter}'); // Hostname -> <current host> (one entry only)
+    cy.contains(hostName).should('be.hidden');
+
+    cy.get('[class="panel-content"]').should('be.visible');
+
+    function select_time_usage_template(count: number = 20) {
+      // selecting a template results in a flaky behavior
+      // TODO: fix flakyness and remove this iteration
```

Comment:
currently preparing a patch that could help you there: we changed the dropdown type back to `AsyncSelect` that displays a small spinner while loading. the absence of this spinner would indicate that the dropdowns are  ready.

Reply:
Ack.